message/validation/validation_test.go

```diff
@@ -1745,38 +1750,7 @@ func Test_ValidateSSVMessage(t *testing.T) {
 
 	// Get error when receiving an SSV message with an invalid signature.
 	t.Run("signature verification", func(t *testing.T) {
-		var afterFork = netCfg.PermissionlessActivationEpoch + 1000
-
-		t.Run("unsigned message before fork", func(t *testing.T) {
-			validator := NewMessageValidator(netCfg, WithNodeStorage(ns)).(*messageValidator)
-
-			validSignedMessage := spectestingutils.TestingProposalMessage(ks.Shares[1], 1)
-
-			encoded, err := validSignedMessage.Encode()
-			require.NoError(t, err)
-
-			message := &spectypes.SSVMessage{
-				MsgType: spectypes.SSVConsensusMsgType,
-				MsgID:   spectypes.NewMsgID(netCfg.Domain, share.ValidatorPubKey, roleAttester),
-				Data:    encoded,
-			}
-
-			encodedMsg, err := commons.EncodeNetworkMsg(message)
-			require.NoError(t, err)
-
-			topicID := commons.ValidatorTopicID(message.GetID().GetPubKey())
-			pMsg := &pubsub.Message{
-				Message: &pspb.Message{
-					Topic: &topicID[0],
-					Data:  encodedMsg,
-				},
-			}
-
-			slot := netCfg.Beacon.FirstSlotAtEpoch(1)
-			receivedAt := netCfg.Beacon.GetSlotStartTime(slot).Add(validator.waitAfterSlotStart(roleAttester))
-			_, _, err = validator.validateP2PMessage(pMsg, receivedAt)
-			require.NoError(t, err)
-		})
+		var afterFork = phase0.Epoch(123456789) + 1000
```

Comment:
We don't differentiate network states anymore, right? So perhaps we don't need names to indicate if it's before or after. Also, what does `+1000` mean?

Reply:
Fixed that too. Thanks